west.yml

```diff
@@ -96,7 +96,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.3.0-rc1
+      revision: pull/191/head
```

Comment:
Update to 360bfac91dae344afde45da8a2580bb269f9f1a1

Reply:
updated